docs/signer-manual.md

```diff
@@ -0,0 +1,58 @@
+# Signer manual for root-signing-staging
+
+## One-time setup
+
+### Yubikey configuration
+
+Generate a PIV Digital Signature key on your hardware key if you don't have one yet.
+
+Using [Yubikey Manager](https://www.yubico.com/support/download/yubikey-manager/)
```

Comment:
If I had found something like  piv-go for python I would have considered it, but I did not. 

Even if had found a library for it... key configuration feels like something I don't really want to do. Since the dongle manufacturer makes a tool that seems decent, I think we can avoid doing it.

One advantage that the script has outside the configuration itself is a level of support for the Yubikey PIV attestations (proofs that the key really is on a Yubikey). I don't personally think this is a major thing:
* tuf-on-ci doesn't support keys on disk so _accidentally_ using non-hardware keys is not an issue
* if we can't trust signers when they say their key is a hardware key, we have bigger problems

I have some ideas how to integrate PIV attestations in tuf-on-ci https://github.com/secure-systems-lab/securesystemslib/issues/623 but I wasn't planning on implementing them unless there's a lot of interest